cmd/manager/main.go

```diff
@@ -83,6 +84,10 @@ func main() {
 		log.Error(err, "unable to add OpenShift apps APIs to scheme")
 		os.Exit(1)
 	}
+	if err := configv1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
Does this mean - the controller will only work on OpenShift 4?

Reply:
Yes.